scripts/tours.js

```diff
@@ -910,17 +911,28 @@ Used for things such as
 - disqualifying/reminding inactive players
 - removing subs */
 function tourStep() {
-    if (tours.paused) { tours.working = !script.battlesStopped; }
+    var systime = parseInt(sys.time(), 10);
+    if (tours.paused) {
+        tours.working = !script.battlesStopped;
+        for (var y1 in tours.tour) {
+            tours.tour[y1].time = systime + pausedTime;
+        }
+    }
     if (script.battlesStopped) {
         tours.paused = true;
+        return;
     } else {
         tours.paused = false;
+        for (var y2 in tours.tour) {
+            if (tours.tour[y2].time > systime) {
+                pausedTime = tours.tour[y2].time - systime;

```

Comment:
you're looping through all the tours, pausedTime is going to be continually overwritten :x